accepted/targeting-packs/targeting-packs.md

```diff
@@ -0,0 +1,149 @@
+# Easy Acquisition of .NET Framework Targeting Packs
+
+**PM** [Immo Landwerth](https://github.com/terrajobst) |
+**Dev** [Daniel Plaisted](https://github.com/dsplaisted)
+
+This proposal is about enabling acquisition of the .NET Framework targeting
+packs via NuGet so that compiling code for a specific version of the .NET
+Framework doesn't require a Windows machine with a machine-wide installation of
+the corresponding targeting pack.
+
+Since .NET Framework 2.0, the .NET platform supported targeting different
+versions of the platform regardless of the .NET Framework version that is
+installed on the developer's machine. This allows developers to target a lower
```

Comment:
I think it wasn't until .NET Framework 4.0 that we got this for real.

Reply:
Hmm I thought 2.0 added TPs but it probably doesn't matter. Will remove the version number :-)

---
In reply to: [174328985](https://github.com/dotnet/designs/pull/33#discussion_r174328985) [](ancestors = 174328985)